tasks/files/snapshot.py

```diff
@@ -900,13 +1453,15 @@ def print_result(rc, message):
         "-s",
         "--suffix",
         dest="suffix",
+        default=None,
```

Comment:
If you use `default=""` then you can use the value directly without needing to check `if args.suffix:` or `if args.suffix is None`

Reply:
I opened an issue: https://github.com/linux-system-roles/snapshot/issues/12